data-prep/README.md

```diff
@@ -15,9 +15,59 @@ Data Cleaning and Munging
 
 A number of cleaning steps are necessary to use the raw data.
 
-### Project Filtering
+### Consumption Data Cleaning
+
+**Elec Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
+* Add leading zeroes to `PREM_ID` where missing (length == 8 or 9) in monthly traces.
+
+* 7702 elec monthly traces with unique SA_ID
+
+**Gas Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
+* Add leading zeroes to `PREM_ID` where missing (length == 8 or 9) in monthly traces.
+
+* 6919 gas monthly traces with unique SA_ID
+
+**Elec Xrefs**
+
+* Add leading zeroes to `char_prem_id` where missing (length == 8 or 9). Appear to have been lost during PGE's export process 
+
+**For Caltrack data analysis: traces with `is_net_metered=Y` should be excluded from analysis.**
```

Comment:
I don't think this is sufficient (nor is it all that was done as far as I understand).  The list of premises that have `is_net_metered==Y` is not a complete record of the premises that we know are net metered.  It is also necessary to check for any premises that have any usage rows with `DIR==R` and add those premises to the list of known net metered premises (which will be excluded from the caltrack analysis)